FAQ.md

```diff
@@ -1,6 +1,43 @@
 # Frequently Asked Questions
 
 These questions come up regularly, so here are the answers.
+# Table of Contents
+
+| [Is this legal?](#is-this-legal)                  |
+| :------------------------------------------------ |
```

Comment:
I'm not sure what these separator lines are about, but I don't think they are needed or desirable.

Reply:
Odd... it was just my editor that did it

Reply:
Header Separator Line (https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables#creating-a-table)